### DISCUSSION TITLE
[Support]: Unable to pull images from ghcr.io

### DISCUSSION THREAD

**Discussion by sochartgit** [blakeblackshear/frigate#10133] [General Support]
### Describe the problem you are having

Unable to pull frigate images from ghcr.io
![image](https://github.com/blakeblackshear/frigate/assets/83184203/19ce2ce7-ddac-4b66-ac73-98ecc7ed42cc)

### Version

3.0

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment]
there is no latest image

**Comment by sochartgit** [comment]
This one is not working neither with a docker pull


***@***.***


De : Nicolas Mowen ***@***.***>
Envoyé : 28 février 2024 09:17
À : blakeblackshear/frigate ***@***.***>
Cc : sochartgit ***@***.***>; Author ***@***.***>
Objet : Re: [blakeblackshear/frigate] [Support]: Unable to pull images from ghcr.io (Discussion #10133)


there is no latest image
